dashboard/app/views/levels/editors/fields/_song_selection.html.haml

```diff
@@ -0,0 +1,9 @@
+%h1.control-legend{data: {toggle: "collapse", target: "#songSelection"}}
+  Limit Song Selection
+
+#songSelection.in.collapse
+  %p
+    Click to
+    %a.select_none{href: '#'} not limit song selection
+  %p (shift-click or cmd-click to select multiple).
+  = f.select :song_selection, @level.class.hoc_songs, {}, {size: 12, multiple: true}
```

Comment:
Is the verbiage here good?

Reply:
I would check with the content editors (maybe in #hoc-2023-activity-dev) and tag AmyBW, Emma Wingreen and Katie Frank. You could also check with them your comment above: https://github.com/code-dot-org/code-dot-org/pull/54208/files#r1357314707